CheatSheets/git-cheatsheet.md

```diff
@@ -68,6 +68,10 @@ created: 2022-10-18
 | `git checkout [file]` | Matches the file with last commit |
 | `git help -a` | Shows the list of all available Git commands |
 | `git restore .` | To restore all files in the current directory |
+| `git-submodule` | Initialize, update or inspect submodules |
```

Comment:
This isn't the correct command. Please, follow this [referral link](https://git-scm.com/book/en/v2/Git-Tools-Submodules) and modify it with the correct command.